### ISSUE TITLE
Consider Node support?

### ISSUE THREAD

**Issue by thibmaek** [Huemul/trae#45]
Might sound like the douchy 'could you build this for me' guy here, but I think this would be great to have in a non browser env (eg. no `window`).

If you want to consider this option I might try and check it out over a PR 😊

**Comment by gillchristian** [comment]
Yes, Node support is on the plans.

We are using [`github's fetch polifyll`](https://github.com/github/fetch) now and it [does not support node env](https://github.com/github/fetch#read-this-first).

Maybe we could switch to [`isomorphic-fetch`](https://github.com/matthew-andrews/isomorphic-fetch) which is built on top of Github's one but supports node. If you want to make a PR, that'll be great :smile:

**Comment by ianaya89** [comment]
Would be great to have node support. We are using [github/fetch](https://github.com/github/fetch) and I think it does not bring support for node.

**Comment by thibmaek** [comment]
Yeah had great results using isomorphic-fetch, let me see what I can do

**Comment by gillchristian** [comment]
Closing in favor of #46 

Thanks @thibmaek !!!

**Comment by joakimbeng** [comment]
@gillchristian 
Great! But I'm afraid it doesn't work...

```bash
❯ node
> trae = require('trae')
ReferenceError: self is not defined
    at /my_project/node_modules/trae/dist/trae.min.js:472:26
    at support.searchParams (/my_project/node_modules/trae/dist/trae.min.js:8:82)
    at Object.<anonymous> (/my_project/node_modules/trae/dist/trae.min.js:11:2)
    at Module._compile (module.js:556:32)
    at Object.Module._extensions..js (module.js:565:10)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
```

This is because the built and minified library is specified in the package.json's `main` field.

**Comment by ndelvalle** [comment]
Thanks for reporting @joakimbeng I'll take a look.

**Comment by ndelvalle** [comment]
@joakimbeng Sorry for taking too long, I've being pretty busy. I added a fix in #55. We didn't create a new release yet, because we want to fix a warning regarding `rollup` and `whatwg-fetch`.
More specific this warning:
```bash
commonjs-proxy:/Users/ndelvalle/Projects/trae/node_modules/whatwg-fetch/fetch.js (1:124) 'default' is not exported by 'node_modules/whatwg-fetch/fetch.js'
```
I hope tomorrow we can solve it and release a new version 🙂

**Comment by ndelvalle** [comment]
#55 has solved this problem.
